duckduckgo/iOS
# Daniel/subscriptions/3.manage subscription

Task/Issue URL: https://app.asana.com/0/72649045549333/1205054784245717/f

**Description**:
- Implement Subscription Management View
- Adds labels to the transaction progress box and reduces the opacity of the background view.

**Steps to test this PR**:
1. Purchase a Subscription
2. Observe that the Progress Dialog now tells you the current Status "Purchasing Subscription or Completing Purchase"
3. Go to Settings -> Manage Subscription
4. Tap `Change Plan or Billing` and observe you're taken to the iOS subscription management page
5. Tap `Remove From this device` and confirm
6. Try to purchase the subscription again.  It should be found and adequately restored
7. Observe that during Restore, the progress dialog says "Restoring subscription."




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
